pkg/cli/environment.go

```diff
@@ -53,9 +55,20 @@ type EnvSettings struct {
 	PluginsDirectory string
 }
 
+var (
+	config     genericclioptions.RESTClientGetter
+	configOnce sync.Once
+
+	// Namespace is the namespace scope.
+	namespace string
```

Comment:
This change makes `namespace` a shared global variable between all EnvSettings objects. Is there a reason this was refactored out of the EnvSettings object?

This should be an internal variable of the EnvSettings object. e.g.

```
// EnvSettings describes all of the environment settings.
type EnvSettings struct {
	// namespace is the namespace scope.
	namespace string
```